staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -217,19 +218,33 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "code", "rejected"})
 
+	webhookSLO := metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_slo_duration_seconds",
+			Help:           "Admission webhook latency histogram in seconds, identified by name and broken out for each operation and for a configurable set of API resources.",
+			Buckets:        []float64{0.005, 0.025, 0.1, 0.5, 1.0, 2.5, 10.0},
```

Comment:
We could use the default buckets instead: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/metrics/histogram.go#L26. The [default values](https://github.com/prometheus/client_golang/blob/2e1c4818ccfdcf953ce399cadad615ff2bed968c/prometheus/histogram.go#L68) are very similar so I think going with the default ones is better for now.

Reply:
Done.